packages/tooling/fast-tooling/src/web-components/html-render/html-render.ts

```diff
@@ -0,0 +1,277 @@
+import { customElement, FASTElement, observable } from "@microsoft/fast-element";
+import { isHTMLElement } from "@microsoft/fast-web-utilities";
+import {
+    htmlMapper,
+    htmlResolver,
+    mapDataDictionary,
+    ResolverConfig,
+} from "../../data-utilities/mapping";
+import {
+    WebComponentDefinition,
+    WebComponentDefinitionTag,
+} from "../../data-utilities/web-component";
+import {
+    DataDictionary,
+    MessageSystem,
+    MessageSystemNavigationTypeAction,
+    MessageSystemType,
+    SchemaDictionary,
+} from "../../message-system";
+import { ActivityType, HTMLRenderLayer } from "../html-render-layer/html-render-layer";
+import { HTMLRenderStyles } from "./html-render.styles";
+import { HTMLRenderTemplate } from "./html-render.template";
+
+@customElement({
+    name: "fast-tooling-html-render",
+    template: HTMLRenderTemplate,
+    styles: HTMLRenderStyles,
+})
+export class HTMLRender extends FASTElement {
+    private dataDictionary: DataDictionary<unknown>;
+
+    private schemaDictionary: SchemaDictionary;
+
+    private navigationConfigId: string = "fast-tooling::html-renderer";
+
+    private dataDictionaryAttr: string = "data-datadictionaryid";
+
+    private tabCounter: number = 1;
+
+    private currentElement: HTMLElement;
+
+    private renderLayers: HTMLRenderLayer[] = [];
+
+    @observable
+    public markup: HTMLElement;
+
+    @observable
+    public markupDefinitions: WebComponentDefinition[] = null;
+
+    @observable
+    public layers: HTMLSlotElement;
+    private layersChanged(oldValue, newValue): void {
+        if (this.$fastController.isConnected) {
+            this.renderLayers = [];
+            if (this.children.length > 0) {
+                Array.from(this.children).forEach((value: HTMLRenderLayer) => {
+                    if (this.isHtmlRenderLayer(value)) {
+                        value.messageSystem = this.messageSystem;
+                        this.renderLayers.push(value);
+                    }
+                });
+            }
+        }
+    }
+
+    @observable
+    public messageSystem: MessageSystem;
+    private messageSystemChanged(): void {
+        if (this.messageSystem !== undefined) {
+            this.tabCounter = 1;
+            this.messageSystem.add({ onMessage: this.handleMessageSystem });
+            this.renderLayers.forEach((value: HTMLRenderLayer) => {
+                value.messageSystem = this.messageSystem;
+            });
+        }
+    }
+
+    connectedCallback() {
```

Comment:
You don't really need the `connectedCallback` if all you're doing is calling `super.connectedCallback()`

Reply:
Removed